x-pack/plugins/fleet/server/services/agents/update_agent_tags_action_runner.ts

```diff
@@ -149,6 +149,9 @@ export async function updateTagsBatch(
   const versionConflictCount = res.version_conflicts ?? 0;
   const versionConflictIds = isLastRetry ? getUuidArray(versionConflictCount) : [];
 
+  const currentNameSpace = soClient.getCurrentNamespace();
+  const namespaces = currentNameSpace ? [currentNameSpace] : [];
```

Comment:
@nchaulet Can you ensure this meets the expectations?

Reply:
Yes that perfect 👍